src/AltinnCore/Designer/Controllers/ManualTestingController.cs

```diff
@@ -85,8 +85,7 @@ public async Task<IActionResult> Index(string org, string service, int reporteeI
                 return LocalRedirect($"/designer/{org}/{service}/ManualTesting/Users/");
             }
 
-            string developer = AuthenticationHelper.GetDeveloperUserName(_httpContextAccessor.HttpContext);
-            _execution.CheckAndUpdateWorkflowFile(org, service, developer);
+            CheckAndUpdateWorkflowFile(org, service);
```

Comment:
applicationOwnerId, applicationId

Reply:
This will require a fix in the cshtml aswell so leaving it as it is
Tobe fixed in:  #1549